LoopKit/InsulinKit/HKQuantitySample+InsulinKit.swift

```diff
@@ -28,9 +37,10 @@ extension HKQuantitySample {
         var metadata: [String: Any] = [
             HKMetadataKeySyncVersion: syncVersion,
             HKMetadataKeySyncIdentifier: syncIdentifier,
-            MetadataKeyHasLoopKitOrigin: true
+            MetadataKeyHasLoopKitOrigin: true,
```

Comment:
[nit] do you want to initialize

`MetadataKeyInsulinType: nil`

here?  No is an okay answer just wanted to mention it...

Reply:
No, all samples LoopKit creates should have this value true, I believe.